pkg/volume/flexvolume/mounter.go

```diff
@@ -94,7 +95,9 @@ func (f *flexVolumeMounter) SetUpAt(dir string, mounterArgs volume.MounterArgs)
 
 	if !f.readOnly {
 		if f.plugin.capabilities.FSGroup {
-			volume.SetVolumeOwnership(f, mounterArgs.FsGroup, mounterArgs.FSGroupChangePolicy, util.FSGroupCompleteHook(f.plugin.GetPluginName()))
+			// fullPluginName helps to distinguish different driver from flex volume plugin
+			fullPluginName := fmt.Sprintf("%s:%s", flexVolumePluginName, f.driverName)
```

Comment:
@msau42 for flex volume, is it ok ?  The format is same as csi plugin.

Reply:
hmmm it seems like the flexvolume name is already qualified with the specific driver. I see metrics formatted with the "flexvolume-" prefix, although I haven't been able to find where in the code that prefix gets added yet.

Reply:
I haven't dug too deeply but driver name includes the prefix from the way plugins are probed. we could just use `drivername` here.

Reply:
Other flex volume metrics use `GetFullQualifiedPluginNameForVolume` too, so I'd prefer to use it in SetVolumeOwnership too.